tests/test_crates_regex.rs

```diff
@@ -0,0 +1,61 @@
+extern crate regex;
+
+///////////////////////////////////////////////////////////////////////
+//                                                                   //
+//                       The Bug Trophy Case                         //
+//                                                                   //
+///////////////////////////////////////////////////////////////////////
```

Comment:
This style of comment isn't anywhere else in the regex crate, and I'm generally not a fan of them. They take up a lot of space, and I'm not really sure what they mean. My guess is that they are just regression tests, and we should probably call them that.